parser/parser.go

```diff
@@ -1599,7 +1606,9 @@ func (p *parser) parseCallOrConversion(fun ast.Expr, isCmd bool) *ast.CallExpr {
 	p.exprLev--
 	var noParenEnd token.Pos
 	if isCmd {
-		noParenEnd = p.pos
+		if !extractTuple {
+			noParenEnd = p.pos
+		}
 	} else {
```

Comment:
maybe we don't need `extractTuple`, just use `else if rparen != token.NoPos`

Reply:
fixed